pyproject.toml

```diff
@@ -12,16 +12,22 @@ packages = [{include = "backend"}]
 
 [tool.poetry.dependencies]
 python = "^3.12"
-fastapi = {extras = ["standard"], version = "^0.114.2"}
+fastapi = {extras = ["standard"], version = "^0.115.0"}
 uvicorn = { extras = ["standard"], version = "^0.30.6" }
 structlog = "^24.4.0"
-sqlalchemy = "^2.0.34"
+sqlalchemy = "^2.0.35"
 psycopg2 = "^2.9.9"
 pydantic-settings = "^2.5.2"
 
+[tool.poetry.scripts]
+openapi = "backend.utils.utils:write_openapi"
```

Comment:
This does not work on pnpm install due to `app` of type `FastApi` not being initialized in runtime created by `openapi.py` script

Reply:
It works perfectly. It runs `utils.py`:
```py
def create_app():
    new_app = FastAPI(title="Landsat Web Tracker API")
    new_app.include_router(index_router, prefix="")
    new_app.include_router(coordinates_router, prefix="/coordinates")

    return new_app


def write_openapi():
    with open('frontend/openapi/api/openapi.json', 'w') as f:
        json.dump(create_app().openapi(), f)
```

It creates minimalist version of app, to just generate openapi.json

Reply:
I see, It is ok if frontend and backend dev is separated but It requires database to be deployed/spined in order to create docs (otherwise backend crashes), idk imo this may be problematic in some edge cases.